### PR TITLE
PMM-12576-change-pwd-not-working-admin

### PR DESCRIPTION
**Pull request by rasika-chivate**
For details, see:

https://jira.percona.com/browse/PMM-12576

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

File: docs/release-notes/2.40.0.md [hunk 1]
```diff
@@ -113,3 +113,16 @@ VictoriaMetrics has been upgraded [v1.93.4](https://docs.victoriametrics.com/CHA
...
+
+    **Solution**: To fix the problem and gain back admin access to the PMM interface execute the following:
+
+    ```sh
+    # psql -U grafana
+    grafana=> update "user" set id='1' where login='admin';
+    UPDATE 1
+    grafana=> \q
+
+    # grafana-cli --homepath=/usr/share/grafana --config=/etc/grafana/grafana.ini admin reset-admin-password <PASS>
```
Comment (rnovikovP) [inline docs/release-notes/2.40.0.md:127]: Maybe link to another section to not duplicate the command?

File: docs/how-to/troubleshoot.md [hunk 2]
```diff
@@ -83,6 +80,21 @@ While PMM is being upgraded, log in to the PMM server and run the following comm
...
+
+**Solution**: To fix the problem and gain back admin access to the PMM interface execute the following:
+
+```sh
+# psql -U grafana
+grafana=> update "user" set id='1' where login='admin';
+UPDATE 1
+grafana=> \q
+
+# grafana-cli --homepath=/usr/share/grafana --config=/etc/grafana/grafana.ini admin reset-admin-password <PASS>
```
Comment (ademidoff) [inline docs/how-to/troubleshoot.md:95]:
```suggestion
# grafana cli --homepath=/usr/share/grafana --config=/etc/grafana/grafana.ini admin reset-admin-password <PASS>
```

Comment (ademidoff) [reply docs/how-to/troubleshoot.md:95]: Please note that the old syntax is deprecated.

File: docs/release-notes/2.40.0.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/release-notes/2.40.0.md]: fixed

**Review by ademidoff — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**